src/backend/main/py.main/caliopen_main/objects/base.py

```diff
@@ -0,0 +1,447 @@
+import zope.interface
+
+from types import *
```

Comment:
no import * be declarative (and you see that types import are not correct on Contact due to this * import